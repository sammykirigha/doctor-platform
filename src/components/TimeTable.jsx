import React from 'react'
import { BsFillClockFill } from 'react-icons/bs';

const TimeTable = () => {
  return (
    <div className='flex flex-row w-full mb-5'>
      <div className=" border shadow-sm rounded-md w-[40%] pt-3">
        <div className='flex flex-row  mx-3 '>
          <span className='flex flex-row items-center '><BsFillClockFill className='text-md text-blue-600  mr-3' /> Monday</span>
          <div className='ml-[140px]  w-80'>Time: <span className='text-blue-400'>8.00 - 20.00</span></div>
        </div>
        <div className='flex flex-row items-center mx-3 mt-3'>
          <span className='flex flex-row items-center '><BsFillClockFill className='text-md text-blue-600 mr-3' /> Tuesday</span>
          <div className='ml-[140px] w-80'>Time: <span className='text-blue-400'>8.00 - 20.00</span></div>
        </div>
        <div className='flex flex-row items-center mx-3 mt-3 '>
          <span className='flex flex-row items-center '><BsFillClockFill className='text-md text-blue-600 mr-3' /> Wednesday</span>
          <div className='ml-[120px] w-80'>Time: <span className='text-blue-400'>8.00 - 20.00</span></div>
        </div>
        <div className='flex flex-row items-center mx-3 mt-3 '>
          <span className='flex flex-row items-center '><BsFillClockFill className='text-md text-blue-600 mr-3' /> Thursday</span>
          <div className='ml-[136px] w-80'>Time: <span className='text-blue-400'>8.00 - 20.00</span></div>
        </div>
        <div className='flex flex-row items-center mx-3 mt-3 '>
          <span className='flex flex-row items-center '><BsFillClockFill className='text-md text-blue-600 mr-3' /> Friday</span>
          <div className='ml-[160px] w-80'>Time: <span className='text-blue-400'>8.00 - 20.00</span></div>
        </div>
        <div className='flex flex-row items-center mx-3 mt-3 '>
          <span className='flex flex-row items-center '><BsFillClockFill className='text-md text-blue-600 mr-3' /> Saturday</span>
          <div className='ml-[140px] w-80'>Time: <span className='text-blue-400'>8.00 - 20.00</span></div>
        </div>
        <div className='flex flex-row items-center mx-3 mt-3 mb-3 '>
          <span className='flex flex-row items-center '><BsFillClockFill className='text-md text-blue-600 mr-3' /> Sunday</span>
          <div className='ml-[150px]'>Time: <span className='text-blue-400'>8.00 - 20.00</span></div>
        </div>
        
      </div>
      <div>right</div>
  </div>
  )
}

export default TimeTable