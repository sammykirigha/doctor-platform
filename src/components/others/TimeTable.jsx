import React from "react";
import { BsFillClockFill, BsTelephone } from "react-icons/bs";
import {HiOutlineMail} from 'react-icons/hi'

const TimeTable = () => {
    return (
        <div className="flex flex-row w-full mb-5">
            <div className=" border shadow-sm rounded-md w-[40%] pt-3">
                <div className="flex flex-row  mx-3 ">
                    <span className="flex flex-row items-center ">
                        <BsFillClockFill className="text-md text-blue-600  mr-3" />{" "}
                        Monday
                    </span>
                    <div className="ml-[140px]  w-80">
                        Time:{" "}
                        <span className="text-blue-400">8.00 - 20.00</span>
                    </div>
                </div>
                <div className="flex flex-row items-center mx-3 mt-3">
                    <span className="flex flex-row items-center ">
                        <BsFillClockFill className="text-md text-blue-600 mr-3" />{" "}
                        Tuesday
                    </span>
                    <div className="ml-[140px] w-80">
                        Time:{" "}
                        <span className="text-blue-400">8.00 - 20.00</span>
                    </div>
                </div>
                <div className="flex flex-row items-center mx-3 mt-3 ">
                    <span className="flex flex-row items-center ">
                        <BsFillClockFill className="text-md text-blue-600 mr-3" />{" "}
                        Wednesday
                    </span>
                    <div className="ml-[118px] w-80">
                        Time:{" "}
                        <span className="text-blue-400">8.00 - 20.00</span>
                    </div>
                </div>
                <div className="flex flex-row items-center mx-3 mt-3 ">
                    <span className="flex flex-row items-center ">
                        <BsFillClockFill className="text-md text-blue-600 mr-3" />{" "}
                        Thursday
                    </span>
                    <div className="ml-[134px] w-80">
                        Time:{" "}
                        <span className="text-blue-400">8.00 - 20.00</span>
                    </div>
                </div>
                <div className="flex flex-row items-center mx-3 mt-3 ">
                    <span className="flex flex-row items-center ">
                        <BsFillClockFill className="text-md text-blue-600 mr-3" />{" "}
                        Friday
                    </span>
                    <div className="ml-[157px] w-80">
                        Time:{" "}
                        <span className="text-blue-400">8.00 - 20.00</span>
                    </div>
                </div>
                <div className="flex flex-row items-center mx-3 mt-3 ">
                    <span className="flex flex-row items-center ">
                        <BsFillClockFill className="text-md text-blue-600 mr-3" />{" "}
                        Saturday
                    </span>
                    <div className="ml-[140px] w-80">
                        Time:{" "}
                        <span className="text-blue-400">8.00 - 20.00</span>
                    </div>
                </div>
                <div className="flex flex-row items-center mx-3 mt-3 mb-3 ">
                    <span className="flex flex-row items-center ">
                        <BsFillClockFill className="text-md text-blue-600 mr-3" />{" "}
                        Sunday
                    </span>
                    <div className="ml-[150px]">
                        Time:{" "}
                        <span className="text-blue-400">8.00 - 20.00</span>
                    </div>
                </div>
            </div>
            <div className="flex flex-row justify-between flex-1 mt-5">
                <div className="ml-5">
                    <div className="flex flex-col ml-32">
                        <span className="h-[60px] w-[60px] bg-blue-100 rounded-md flex items-center justify-center">
                            <BsTelephone className="text-2xl text-blue-800 font-bold" />
                        </span>
                        <span className="text-lg text-slate-900 font-bold hover:text-blue-600 mt-4">
                            Phone
                        </span>
                    </div>

                    <div className="flex flex-col justify-center">
                        <p className="text-lg text-slate-900 opacity-40">
                            Great doctor if you need your family member to get
                            effective immediate assistance
                        </p>
                        <span className="text-md text-blue-500 ml-20 mt-5">
                            +152 534-468-854
                        </span>
                    </div>
                </div>

                <div className="ml-5">
                    <div className="flex flex-col ml-32">
                        <span className="h-[60px] w-[60px] bg-blue-100 rounded-md flex items-center justify-center">
                            <HiOutlineMail className="text-2xl text-blue-800 font-bold" />
                        </span>
                        <span className="text-lg text-slate-900 font-bold hover:text-blue-600 mt-4">
                            Email
                        </span>
                    </div>

                    <div className="flex flex-col justify-center">
                        <p className="text-lg text-slate-900 opacity-40">
                            Great doctor if you need your family member to get effective immediate assistance
                        </p>
                        <span className="text-md text-blue-500 ml-20 mt-5">
                            contact@example.com
                        </span>
                    </div>
                </div>
            </div>
        </div>
    );
};

export default TimeTable;
