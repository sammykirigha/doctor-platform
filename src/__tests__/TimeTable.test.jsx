import { cleanup, render, screen } from '@testing-library/react';
import TimeTable from '../components/TimeTable'


describe('The suit is to test the body component', () => {

	it('Snapshot of TimeTable', () => {
		const { asFragment } = render(<TimeTable />)
		expect(asFragment()).toMatchSnapshot()
	})

	it('should render the TimeTable with the contact@example.com text', () => {
		render(<TimeTable />)
		let element = screen.getByText('contact@example.com');

		expect(element.textContent).toBe('contact@example.com')
	})

	it('should render the TimeTable with the right text', () => {
		render(<TimeTable />)
		let element = screen.getByText( 'Phone');

		expect(element.textContent).toBe('Phone')
	})

})