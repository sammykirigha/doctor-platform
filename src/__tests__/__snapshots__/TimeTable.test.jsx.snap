// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The suit is to test the body component Snapshot of TimeTable 1`] = `
<DocumentFragment>
  <div
    class="flex flex-row w-full mb-5"
  >
    <div
      class=" border shadow-sm rounded-md w-[40%] pt-3"
    >
      <div
        class="flex flex-row  mx-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600  mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Monday
        </span>
        <div
          class="ml-[140px]  w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3"
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Tuesday
        </span>
        <div
          class="ml-[140px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Wednesday
        </span>
        <div
          class="ml-[118px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Thursday
        </span>
        <div
          class="ml-[134px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Friday
        </span>
        <div
          class="ml-[157px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Saturday
        </span>
        <div
          class="ml-[140px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 mb-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Sunday
        </span>
        <div
          class="ml-[150px]"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
    </div>
    <div
      class="flex flex-row justify-between flex-1 mt-5"
    >
      <div
        class="ml-5"
      >
        <div
          class="flex flex-col ml-32"
        >
          <span
            class="h-[60px] w-[60px] bg-blue-100 rounded-md flex items-center justify-center"
          >
            <svg
              class="text-2xl text-blue-800 font-bold"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3.654 1.328a.678.678 0 0 0-1.015-.063L1.605 2.3c-.483.484-.661 1.169-.45 1.77a17.568 17.568 0 0 0 4.168 6.608 17.569 17.569 0 0 0 6.608 4.168c.601.211 1.286.033 1.77-.45l1.034-1.034a.678.678 0 0 0-.063-1.015l-2.307-1.794a.678.678 0 0 0-.58-.122l-2.19.547a1.745 1.745 0 0 1-1.657-.459L5.482 8.062a1.745 1.745 0 0 1-.46-1.657l.548-2.19a.678.678 0 0 0-.122-.58L3.654 1.328zM1.884.511a1.745 1.745 0 0 1 2.612.163L6.29 2.98c.329.423.445.974.315 1.494l-.547 2.19a.678.678 0 0 0 .178.643l2.457 2.457a.678.678 0 0 0 .644.178l2.189-.547a1.745 1.745 0 0 1 1.494.315l2.306 1.794c.829.645.905 1.87.163 2.611l-1.034 1.034c-.74.74-1.846 1.065-2.877.702a18.634 18.634 0 0 1-7.01-4.42 18.634 18.634 0 0 1-4.42-7.009c-.362-1.03-.037-2.137.703-2.877L1.885.511z"
              />
            </svg>
          </span>
          <span
            class="text-lg text-slate-900 font-bold hover:text-blue-600 mt-4"
          >
            Phone
          </span>
        </div>
        <div
          class="flex flex-col justify-center"
        >
          <p
            class="text-lg text-slate-900 opacity-40"
          >
            Great doctor if you need your family member to get effective immediate assistance
          </p>
          <span
            class="text-md text-blue-500 ml-20 mt-5"
          >
            +152 534-468-854
          </span>
        </div>
      </div>
      <div
        class="ml-5"
      >
        <div
          class="flex flex-col ml-32"
        >
          <span
            class="h-[60px] w-[60px] bg-blue-100 rounded-md flex items-center justify-center"
          >
            <svg
              class="text-2xl text-blue-800 font-bold"
              fill="none"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3 8l7.89 5.26a2 2 0 002.22 0L21 8M5 19h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v10a2 2 0 002 2z"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
          </span>
          <span
            class="text-lg text-slate-900 font-bold hover:text-blue-600 mt-4"
          >
            Email
          </span>
        </div>
        <div
          class="flex flex-col justify-center"
        >
          <p
            class="text-lg text-slate-900 opacity-40"
          >
            Great doctor if you need your family member to get effective immediate assistance
          </p>
          <span
            class="text-md text-blue-500 ml-20 mt-5"
          >
            contact@example.com
          </span>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`The suit is to test the body component Snapshot of UserMessage 1`] = `
<DocumentFragment>
  <div
    class="flex flex-row w-full mb-5"
  >
    <div
      class=" border shadow-sm rounded-md w-[40%] pt-3"
    >
      <div
        class="flex flex-row  mx-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600  mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Monday
        </span>
        <div
          class="ml-[140px]  w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3"
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Tuesday
        </span>
        <div
          class="ml-[140px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Wednesday
        </span>
        <div
          class="ml-[118px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Thursday
        </span>
        <div
          class="ml-[134px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Friday
        </span>
        <div
          class="ml-[157px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Saturday
        </span>
        <div
          class="ml-[140px] w-80"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
      <div
        class="flex flex-row items-center mx-3 mt-3 mb-3 "
      >
        <span
          class="flex flex-row items-center "
        >
          <svg
            class="text-md text-blue-600 mr-3"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 3.5a.5.5 0 0 0-1 0V9a.5.5 0 0 0 .252.434l3.5 2a.5.5 0 0 0 .496-.868L8 8.71V3.5z"
            />
          </svg>
           Sunday
        </span>
        <div
          class="ml-[150px]"
        >
          Time: 
          <span
            class="text-blue-400"
          >
            8.00 - 20.00
          </span>
        </div>
      </div>
    </div>
    <div
      class="flex flex-row justify-between flex-1 mt-5"
    >
      <div
        class="ml-5"
      >
        <div
          class="flex flex-col ml-32"
        >
          <span
            class="h-[60px] w-[60px] bg-blue-100 rounded-md flex items-center justify-center"
          >
            <svg
              class="text-2xl text-blue-800 font-bold"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3.654 1.328a.678.678 0 0 0-1.015-.063L1.605 2.3c-.483.484-.661 1.169-.45 1.77a17.568 17.568 0 0 0 4.168 6.608 17.569 17.569 0 0 0 6.608 4.168c.601.211 1.286.033 1.77-.45l1.034-1.034a.678.678 0 0 0-.063-1.015l-2.307-1.794a.678.678 0 0 0-.58-.122l-2.19.547a1.745 1.745 0 0 1-1.657-.459L5.482 8.062a1.745 1.745 0 0 1-.46-1.657l.548-2.19a.678.678 0 0 0-.122-.58L3.654 1.328zM1.884.511a1.745 1.745 0 0 1 2.612.163L6.29 2.98c.329.423.445.974.315 1.494l-.547 2.19a.678.678 0 0 0 .178.643l2.457 2.457a.678.678 0 0 0 .644.178l2.189-.547a1.745 1.745 0 0 1 1.494.315l2.306 1.794c.829.645.905 1.87.163 2.611l-1.034 1.034c-.74.74-1.846 1.065-2.877.702a18.634 18.634 0 0 1-7.01-4.42 18.634 18.634 0 0 1-4.42-7.009c-.362-1.03-.037-2.137.703-2.877L1.885.511z"
              />
            </svg>
          </span>
          <span
            class="text-lg text-slate-900 font-bold hover:text-blue-600 mt-4"
          >
            Phone
          </span>
        </div>
        <div
          class="flex flex-col justify-center"
        >
          <p
            class="text-lg text-slate-900 opacity-40"
          >
            Great doctor if you need your family member to get effective immediate assistance
          </p>
          <span
            class="text-md text-blue-500 ml-20 mt-5"
          >
            +152 534-468-854
          </span>
        </div>
      </div>
      <div
        class="ml-5"
      >
        <div
          class="flex flex-col ml-32"
        >
          <span
            class="h-[60px] w-[60px] bg-blue-100 rounded-md flex items-center justify-center"
          >
            <svg
              class="text-2xl text-blue-800 font-bold"
              fill="none"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3 8l7.89 5.26a2 2 0 002.22 0L21 8M5 19h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v10a2 2 0 002 2z"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
          </span>
          <span
            class="text-lg text-slate-900 font-bold hover:text-blue-600 mt-4"
          >
            Email
          </span>
        </div>
        <div
          class="flex flex-col justify-center"
        >
          <p
            class="text-lg text-slate-900 opacity-40"
          >
            Great doctor if you need your family member to get effective immediate assistance
          </p>
          <span
            class="text-md text-blue-500 ml-20 mt-5"
          >
            contact@example.com
          </span>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
